frontend/.changeset/eighty-bottles-hope.md

```diff
@@ -0,0 +1,6 @@
+---
+"@liam-hq/erd-core": patch
+"@liam-hq/cli": patch
+---
+
+📈 Add cliVer parameter to log event functions and component.s
```

Comment:
nits

```suggestion
📈 Add cliVer parameter to log event functions and components
```

Reply:
Oh, sorry🤦‍♀️. I made mistake...

Reply:
oh!!
I wonder if I can rewrite the changeset later. I'll give it a try.